### PR TITLE
fix for worker dying issue when returned keys do not contain '/'

### PR DESCRIPTION
**Pull request by senthilkumar-kj**

### CODE REVIEW

**Comment by xiaokai-wang** [comment]
thanks

File: src/ngx_stream_upsync_module.c [hunk 1]
```diff
@@ -1155,7 +1155,7 @@ ngx_stream_upsync_consul_parse_json(void *data)
         cJSON *temp1 = cJSON_GetObjectItem(server_next, "Key");
         if (temp1 != NULL && temp1->valuestring != NULL) {
             p = (u_char *)ngx_strrchr(temp1->valuestring, '/');
```
Comment (xiaokai-wang) [inline src/ngx_stream_upsync_module.c]: (p == NULL) doesnt mean (temp1->valuestring) is null, we should give some message why that is not work, is this better?

File: src/ngx_stream_upsync_module.c [hunk 2]
```diff
@@ -1155,7 +1155,7 @@ ngx_stream_upsync_consul_parse_json(void *data)
...
-            if (ngx_stream_upsync_check_key(p) != NGX_OK) {
```
Comment (xiaokai-wang) [inline src/ngx_stream_upsync_module.c]: Like: 
if (p == NULL) {
     ngx_log_error(NGX_LOG_ERR, ngx_cycle->log, 0, "upsync_parse_json: %s format is illege", 
                             temp1->valuestring)
}
or something.

Are you interested to make a pr again? thanks.

**Comment by xiaokai-wang** [comment]
@senthilkumar-kj please make a new pr, then sync to nginx-upsync-module.

thanks.

**Comment by senthilkumar-kj** [comment]
@xiaokai-wang yes, I'll submit another PR today.

File: src/ngx_stream_upsync_module.c (hunk 2 above)
Comment (senthilkumar-kj) [inline src/ngx_stream_upsync_module.c]: In that case, can we move this check to ngx_stream_upsync_check_key() as well? That's where other loggers are and check/validations are done.

Comment (xiaokai-wang) [inline src/ngx_stream_upsync_module.c]: Maybe that is much better, thanks @senthilkumar-kj .

**Pull request merged**